.changeset/brave-pots-drop.md

```diff
@@ -0,0 +1,18 @@
+---
+'astro': minor
+---
+
+Partials
+
+Any page component can now be identified as a *partial*, allowing you fetch them in the client in order to replace only parts of the page. Partials are used in conjuction with a rendering library, like htmx or Stimulus or even just jQuery.
+
+Pages marked as partials do not have a `doctype` or any head content included in the rendered result. You can mark any page as a partial by setting this option:
```

Comment:
```suggestion
You can mark any `.astro` page as a partial with an export statement:
```

Reply:
Do you not think it's worth mentioning what this feature does here? I'm thinking in the context of the blog post, shouldn't we explain that it removes the head content?

Reply:
I'm looking now and one of my comments (my main one) is entirely missing! I rewrote a bunch of stuff to add this earlier!